docs/UserGuide.md

```diff
@@ -309,6 +342,17 @@ Adds an Event to Managera.
 
 Format: `addEvent n/NAME d/DATE [t/TIME]` 
 
+* The event name must contain only alphanumeric characters.
+* The event date must be given in YYYY-MM-DD format.
+* The event time must be given in 24-hr format e.g., 0000 to 2359.
+  
+
+* Managera cannot accept duplicate events. An event is considered duplicate if all of its attributes
+  (name, date, time) are identical with those of an existing event.
```

Comment:
I think our current implementation checks for name and date only. Should we modify to check time too?